app/models/host.rb

```diff
@@ -3,7 +3,7 @@
 # Host representation in insights compliance backend. Most of the times
 # these hosts will also show up in the insights-platform host inventory.
 class Host < ApplicationRecord
-  scoped_search on: %i[id name account_id os_major_version os_minor_version]
```

Comment:
scoped_search on id isn't something other models allow. I'm not quite sure why it'd be useful when there's a `show` endpoint. There wouldn't ever be a reason to use a 'like' search (`id ~ foo`) on UUIDs :shrug: